WordPress/Classes/ViewRelated/Site Creation/Shared/TableViewOffsetCoordinator.swift

```diff
@@ -15,20 +15,37 @@ final class TableViewOffsetCoordinator {
     /// The table view to coordinate
     private weak var tableView: UITableView?
 
+    //// The view containing the toolbar
+    private weak var container: UIView?
+
+    //// The toolbar
+    private weak var toolbar: UIView?
+
+    //// The constraint linking the bottom of the toolbar to its container
+    private weak var toolbarBottomConstraint: NSLayoutConstraint?
+
     /// The value of the bottom constraint constant is set in response to the keyboard appearance
```

Comment:
I created this errant comment! 🙈  I can tidy that up at the same time as the removeObserver call removal if you'd like.